content/_data/upgrades/2-249-1.adoc

```diff
@@ -59,3 +59,121 @@ the following extra upgrade steps are required:
 2. Upgrade agents with .NET Framework 4.0+ by downloading the recent Windows Service Wrapper 2.x
   version from link:https://github.com/winsw/winsw/releases[WinSW GitHub Releases]
   and manually replacing the wrapper ".exe" files in the agent workspaces.
+
+==== View statusFilter removed from configuration as code
+
+As part of a performance improvement for list views, the `statusFilter` key is now an optional, separate item in the Jenkins view configuration.
+The `statusFilter` key in link:https://plugins.jenkins.io/configuration-as-code/[configuration as code] view definitions will need to be removed or replaced with a new entry as part of the upgrade to Jenkins 2.249.1.
+
+Previous configuration::
+The previous configuration looked like this:
++
+----
+  views:
+  - list:
+      columns:
+      - "status"
+      - "weather"
+      - "jobName"
+      description: "Jobs failing recently"
+      jobFilters:
+      - jobStatusFilter:
+          aborted: true
+          failed: true
+          includeExcludeTypeString: "includeMatched"
+      name: "Failed Jobs"
+      statusFilter: true # This must be removed
+----
+
+New configuration::
+The new configuration looks like this:
++
+----
+  views:
+  - list:
+      columns:
+      - "status"
+      - "weather"
+      - "jobName"
+      description: "Jobs failing recently"
+      jobFilters:
+      - jobStatusFilter:
+          aborted: true
+          failed: true
+          includeExcludeTypeString: "includeMatched"
+      - statusFilter: # This is the replacement section
+          statusFilter: true
+      name: "Failed Jobs"
+----
+
+If the  `statusFilter` key is not updated or removed from the YAML file, Jenkins halts during startup with a stack trace.
+The initial section of the failure stack trace looks like this:
+
+.Configuration as code stack trace
+----
+SEVERE  jenkins.InitReactorRunner$1#onTaskFailed: Failed ConfigurationAsCode.init
+io.jenkins.plugins.casc.ConfiguratorException: 'statusFilter' is deprecated
+        at io.jenkins.plugins.casc.BaseConfigurator.configure(BaseConfigurator.java:321)
+        at io.jenkins.plugins.casc.BaseConfigurator.configure(BaseConfigurator.java:270)
+        ...
+----
+
+==== Alternate URL removed from inbound agent
+
+Jenkins 2.249.1 no longer provides a fallback URL in the inbound agent launch file (the "slave-agent.jnlp" file).
+Prior releases allowed the administrator to configure a second URL for inbound agents that was different from the Jenkins root URL.
```

Comment:
Not as such. Rather, if the `slave-agent.jnlp` was _loaded_ from a URL whose prefix was distinct from the configured Jenkins root URL, that would be added as a fallback.

Reply:
Thanks.  Updated the text in 8270c09cf8bd70993b9e511c39a1839efd6378ba @jglick